### ISSUE TITLE
SMTP Client 11 (smtpclient-11.tac) throws AttributeError exception

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5686]
|[<img alt="argonemyth's avatar" src="https://avatars.githubusercontent.com/u/965739?s=50" width="50" height="50">](https://github.com/argonemyth)| @argonemyth reported|
|-|-|
|Trac ID|trac#5686|
|Type|defect|
|Created|2012-05-24 03:05:16Z|

Well, that's because a mxRecord instance no longer has the 'exchange' attribute - should replace 'mxRecord.exchange' with 'mxRecord.name'. 

It will be quick. Let me fix it first (before updating the whole SMTP client tutorial), so people can have a working SMTP client. 

Attachments:

* [smtpclient11-fix-20120524.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/042/0429d6b477c3d9d3fbe6229438d8d6d08a8cb413/f146a50b68f542d26a55f359d8167ffb9c13957e.patch) (1008 bytes) - added by argonemyth on 2012-05-24 03:18:41Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5686 5686
type__defect defect
reporter__argonemyth argonemyth
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__documentation_tutorial_easy documentation tutorial easy
time__1337828716000000 1337828716000000
changetime__1337952240000000 1337952240000000
version__None None
owner__itamar itamar
cc__jesstess cc__argonemyth
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="argonemyth's avatar" src="https://avatars.githubusercontent.com/u/965739?s=50" width="50" height="50">](https://github.com/argonemyth)<a name="note_1"></a>|@argonemyth removed owner|
|-|-|

Updated the tac and doc files (see patch mtpclient11-fix-20120524.patch), smtpclient-11.tac should be a working client now.

(Oops, forgot to change the component when I created the ticket!)

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_2"></a>|@itamarst set owner to @itamarst|
|-|-|

Looks good; I'll merge.

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_3"></a>|@itamarst set status to `closed`|
|-|-|

(In [34433]) Apply 5686 patch: Use correct attribute in MXCalculator example.

Author: argonemyth
Review: itamar
Fixes: [#5686](https://github.com/twisted/twisted/issues/5686)
